crates/nu-plugin-test-support/src/plugin_test.rs

```diff
@@ -337,12 +337,6 @@ impl PluginTest {
                 // All equal, and same length
                 Ok(true)
             }
-            (Value::Range { val: a_rng, .. }, Value::Range { val: b_rng, .. }) => {
```

Comment:
seems like an odd thing to remove for this pr?

Reply:
Somehow my other range PR was affecting the CI for this PR. I merged with main and fixed a compile error that was somehow missed? I think the CI was drunk, it also failed to "download bash" for one of the runs